.licenserc.yaml

```diff
@@ -30,6 +30,7 @@ header:
     - 'CODEOWNERS'
     - 'icon.svg'
     - 'LICENSE'
+    - 'tests/integration/provider_charm/**'
```

Comment:
Why do you need to exclude this?

Reply:
That folder's content is taken literally from https://github.com/canonical/tls-certificates-interface/tree/main/tests/unit/charms/tls_certificates_interface/v2/dummy_provider_charm, it is not of my making. So I'd like to conserve the code as similar to the source as possible.

Reply:
Isn't there a charm implementing the relation you can just deploy? You are duplicating code across repositories. If not, you can also use AnyCharm